frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -104,9 +134,187 @@ describe("WMG tissue auto-expand", () => {
     await checkTissues(page);
     await removeCellFilter(page);
     await checkTissues(page, ["lung"], []);
-    await checkCellTypes(page, cells);
+    await checkElementVisible(page, cells, CELL_TYPE_TEST_ID);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Sex filter selected
+   * Filter Female from Sex filter. Filter Abdomen and Blood from the Tissue
+   * filter - those should be expanded, Add Cell type filtering,
+   * only B Cell type should appear and expanded.
+   */
+  test("Tissue auto expansion - cross filter with Sex filter selected", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await filterSelection(
+      page,
+      SEX_FILTER_TEST_ID,
+      SEX_FILTER_LABEL,
+      SEX_FILTER_SELECTION
+    );
+    await filterTissues(page, EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER);
+    await filterCellType(page, 1);
+    await checkTissues(page, EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER);
+    await checkElementVisible(page, EXPECTED_VISIBLE_CELL, CELL_TYPE_TEST_ID);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Sex filter, check expansion
+   * Filter 3 Tissues ["abdomen", "axilla", "blood"]. Collapse Abdomen. Select
+   * Female from the Sex filter. Tissue filter should now only have Abdomen and
+   * Blood selected. Only Abdomen and Blood should be visible. Abdomen should
+   * remain collapsed. Remove Sex filter. Tissue filter should now only have
+   * Abdomen and Blood selected. Only Abdomen and Blood should be visible.
+   * Abdomen should remain collapsed.
+   */
+  test("Tissue auto expansion - cross filter with Sex filter, check expansion", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await filterTissues(page, FILTERED_TISSUES);
+    await collapseTissue(page, FILTERED_TISSUES[0]);
+    await filterSelection(
+      page,
+      SEX_FILTER_TEST_ID,
+      SEX_FILTER_LABEL,
+      SEX_FILTER_SELECTION
+    );
+    await expect(page.getByTestId(TISSUE_NODE_TEST_ID)).toHaveCount(
+      EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER.length
+    );
+    await checkTissues(
+      page,
+      EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER,
+      EXPECTED_EXPANDED_TISSUES
+    );
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Publication filter selected
+   * Filter 'Ahren et al. Cell 2022' from the Publication filter. Filter Blood
+   * from the tissue filter. Blood should be expanded. Collapse Blood. add Cell
+   * filter for B Cell. Only B Cell should appear and Blood should be expanded.
+   */
+  test("Tissue auto expansion - cross filter with Publication filter selected", async ({
+    page,
+  }) => {
+    const tissues = FILTERED_TISSUES.slice(2, 3);
```

Comment:
Maybe let's just hard code blood here for readability, especially since the comment is already specifying `blood`!

`const tissues = ["blood"];`